modules/core/sfx.js

```diff
@@ -119,9 +122,16 @@ yargs(hideBin(process.argv))
           description:
             'When error occurs during SSR, SFX displays internal server error instead of falling back to client-only rendering.',
         })
+        .option('require', {
+          alias: 'r',
+          type: 'string',
+          description: 'Module to be imported before all the other scripts.',
+        })
     },
     handler: async (argv) => {
       try {
+        if (argv.require) await import(argv.require)
```

Comment:
Can you make it so that SFX supports multiple requires? Is it even possible in yargs?